dagshub/upload/wrapper.py

```diff
@@ -427,7 +428,7 @@ def add_dir(self, local_path, glob_exclude=""):
         for root, dirs, files in os.walk(local_path):
```

Comment:
Could you replace `os.walk` here with something with `posixpath`?

Reply:
Doesn't look like it's possible, but maybe I can cast the path to a posixpath

Reply:
COOL, MERGE IT

Reply:
nvm I already return a posixpath at that point